lib/src/ast/attribute.dart

```diff
@@ -55,33 +50,51 @@ abstract class AttributeAst implements TemplateAst {
   /// Static attribute value; may be `null` to have no value.
   String get value;
 
+  /// Static attribute value with quotes attached;
+  /// may be `null` to have no value.
+  String get quotedValue;
+
   @override
   String toString() {
     if (value != null) {
-      return '$AttributeAst {$name="$value"}';
+      return '$AttributeAst {$name=$quotedValue}';
     }
     return '$AttributeAst {$name}';
   }
 }
 
-class _ParsedAttributeAst extends TemplateAst with AttributeAst {
-  final NgToken _nameToken;
-  final NgToken _valueToken;
+class ParsedAttributeAst extends TemplateAst with AttributeAst, OffsetInfo {
```

Comment:
Here and elsewhere in the PR - can we get a 1-line document for all the ParsedXXX clases, and their public properties (like `nameToken`, below)?

I want to make sure the generated dartdoc isn't too mysterious for new users - even if it's obvious enough to us.